habitat/plan.sh

```diff
@@ -0,0 +1,171 @@
+pkg_name=chef-workstation
+pkg_origin=chef
+pkg_maintainer="The Chef Maintainers <humans@chef.io>"
+pkg_description="Chef Workstation - Opinionated tools for getting the most out of the Chef ecosystem"
+pkg_license=('Apache-2.0')
+pkg_bin_dirs=(bin)
+pkg_build_deps=(
+  core/make
+  core/gcc
+  core/gcc-libs
+  core/coreutils
+  core/pkg-config
+)
+
+
+ruby_pkg="core/ruby26"
+RUBY_MAJOR_MINOR_VERSION=2.6
+RUBY_PATCH_VERSION=3
+
+RUBY_VERSION=$RUBY_MAJOR_MINOR_VERSION.$RUBY_PATCH_VERSION
+RUBYGEM_VERSION=$RUBY_MAJOR_MINOR_VERSION.0
+
+pkg_deps=(
+  core/glibc
+  core/busybox-static
+  ${ruby_pkg}
+  core/libxml2
+  core/libxslt
+  core/xz
+  core/zlib
+  core/bundler
+  core/openssl
+  core/cacerts
+  core/libffi
+  core/libarchive
+  core/git
+)
+
+pkg_svc_user=root
+
+pkg_version() {
+  cat /src/VERSION
+}
+
+do_before() {
+  do_default_before
+  update_pkg_version
+}
+
+do_verify() {
+  return 0
+}
+
+
+
+do_prepare() {
+  export OPENSSL_LIB_DIR=$(pkg_path_for openssl)/lib
+  export OPENSSL_INCLUDE_DIR=$(pkg_path_for openssl)/include
+  export SSL_CERT_FILE=$(pkg_path_for cacerts)/ssl/cert.pem
+
+  build_line "Setting link for /usr/bin/env to 'coreutils'"
+  [[ ! -f /usr/bin/env ]] && ln -s $(pkg_path_for coreutils)/bin/env /usr/bin/env
+
+  return 0
```

Comment:
I don't think you want this return since you are doing something in this callback